### PR TITLE
Develop vighneshwar

### PR DESCRIPTION
**Pull request by pliyo**
Removing if else

### CODE REVIEW

File: src/GildedRose.Console/Program.cs [hunk 1]
```diff
@@ -27,9 +27,7 @@ static void Main(string[] args)
                                           }
 
                           };
-
-            app.UpdateQuality();
```
Comment (pliyo) [inline src/GildedRose.Console/Program.cs:31]: Why did you remove this line?

File: src/GildedRose.Console/Program.cs [hunk 2]
```diff
@@ -38,43 +36,6 @@ public void UpdateQuality()
         {
             for (var i = 0; i < Items.Count; i++)
             {
-                if (Items[i].Name != "Aged Brie" && Items[i].Name != "Backstage passes to a TAFKAL80ETC concert")
```
Comment (pliyo) [inline src/GildedRose.Console/Program.cs:41]: Nice!

Comment (Pauljgraham) [reply src/GildedRose.Console/Program.cs:41]: mY COMMENT
